helmfile/values/metrics-server.yaml.gotmpl

```diff
@@ -2,9 +2,22 @@ rbac:
   create: true
   pspEnabled: true
 
+# added just to keep only InternalIP as kubelet preferred address
+defaultArgs:
+  - --cert-dir=/tmp
+  - --kubelet-preferred-address-types=InternalIP
+  - --kubelet-use-node-status-port
+  - --metric-resolution=15s
```

Comment:
I think we scrape most (or all) endpoints every 30 seconds. Any reason why this is 15s?

Reply:
I just kept the default value.
I will change it to 30s, but this is not he same as the Prometheus scrape interval. This value is used by the metrics server in the autoscalling process.

Reply:
Ok, then I think this is ok to keep at 15s if you want. You can choose.